src/hooks/useCalculation.ts

```diff
@@ -0,0 +1,115 @@
+import { useEffect, useState } from 'react';
+import type { UnitBoxes } from '../types';
+import { useStateValue } from '../state/state';
+import { ARDataPriceEstimator } from '../utils';
+
+/** ARDataPriceEstimator instance, will fire off initial fetch calls when constructed */
+const arDataPriceEstimator = new ARDataPriceEstimator();
+
+const winstonToArValue = 0.000_000_000_001;
```

Comment:
We're seeing so much code generated around the codebases that deals with unit conversion. I'd like to see this all end up in Core as a set of types. I could see us removing lots of duplicated code that checks Winston and byte count volumes and just using types that are verified at construction time. E.g. rather than accepting a number for a winston count, we'd accept a type like ARToken which has getters for AR value or Winston value and which, at construction time, dealt with validity. It'd clean up a lot of code.

Reply:
Also, please rename to `arPerWinston`.